src/devices/AirConditioner.ts

```diff
@@ -663,7 +663,7 @@ export default class AirConditioner extends baseDevice {
         type: ValueType.Range,
         min: 0,
         max: 0,
-        step: 0.01,
+        step: 1,
```

Comment:
if we use `1` in step, some AC with 0.5 step not worked correcly, and fahrenheit unit not worked too

Reply:
Anyway to detect the step from AC model json? I looked but I couldn't find it.
if not then it could be added in a map for each model.

Reply:
> Anyway to detect the step from AC model json? I looked but I couldn't find it.

I couldn't found anything too

> If not then it could be added in a map for each model.

I tried this way before, but somehow, I got some device same model number but different step (as I remember)

> As for Fahrenheit, step is still one my devices.

in Home app, you can adjust fahrenheit temperature in step 1 by 1, but Homekit always handle it in celcius, it's mean they will convert fahrenheit value to celcius and send it to server

btw, I have config for AC `ac_temperature_unit`, we can use this config value to change this step value I think

Reply:
Would you prefer if I remove the stepping change for now and proceed with the PR and create a dedicated PR for step changes?
It seems that it will require a better look to make sure it's compatible with wide range of devices and to make sure that temp conversion handling is correct.

Reply:
I was mistaken about the Fahrenheit unit, a step of 1 or 0.01 works perfectly with Fahrenheit, but not with Celsius, for example people can't set temperature to 23.5 *C even their AC supported that range
so yes, we need revert it to step `0.01` to make sure it's compatible

what problem do you have about that step? we can discuss more
I tried to change temperature display unit in Home app to Fahrenheit and everything still look good even with step `0.01`